runtime/commonMain/src/kotlinx/serialization/json/internal/StreamingJsonOutput.kt

```diff
@@ -147,15 +147,15 @@ internal class StreamingJsonOutput(private val composer: Composer, override val
 
     override fun encodeFloat(value: Float) {
         if (configuration.strictMode && !value.isFinite()) {
-            throw JsonInvalidValueInStrictModeException(value)
```

Comment:
You can make `JsonInvalidValueInStrictModeException` constructor-like function instead of copy-pasting exception text